### PR TITLE
Test the cell creator buttons

### PR DESCRIPTION
**Pull request by jdfreder**
Woot

### CODE REVIEW

**Comment by rgbkrk** [comment]
These tests should end in `_spec`, not a big deal though.

File: test/renderer/components/cell/cell-creator-buttons.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    });
+  });
+  it('can create code cell', () => {
+    return new Promise(resolve => {
+      const context = {
+        dispatch: action => {
+          expect(action.id).to.equal('test');
+          expect(action.cellType).to.equal('code');
+          expect(action.type).to.equal(NEW_CELL_AFTER);
+          resolve();
```
Comment (rgbkrk) [inline test/renderer/components/cell/cell-creator-buttons.js:55]: NICE!

File: test/renderer/components/cell/cell-creator-buttons.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+          expect(action.id).to.equal('test');
+          expect(action.cellType).to.equal('markdown');
+          expect(action.type).to.equal(NEW_CELL_AFTER);
+          resolve();
+        }
+      };
+      const component = shallow(
+        <CellCreatorButtons above={false} id='test' />
+      , { context });
+      component.find('button.add-text-cell').simulate('click');
```
Comment (rgbkrk) [inline test/renderer/components/cell/cell-creator-buttons.js:45]: Pretty slick here.

**Comment by rgbkrk** [comment]
:heart: for enzyme

**Comment by jdfreder** [comment]
Decrease of coverage ??????????? How's that possible!?

**Comment by rgbkrk** [comment]
I am confused too.

**Comment by rgbkrk** [comment]
Hmmm, the coverage diff only shows the introduction of the new class. I wonder if codecov doesn't know how to deal with JSX.

**Comment by rgbkrk** [comment]
Merging because obviously this is more than before. :smile:

**Comment by jdfreder** [comment]
I'm surprised the create cell above function, the one that sits behind the store, isn't newly covered because of this...

**Comment by jdfreder** [comment]
oh aha, it's because of my spoofed dispatch

**Pull request merged**